main/proxy.go

```diff
@@ -49,7 +50,7 @@ func main() {
 
 	// show version number
 	if *version {
-		fmt.Printf("OpenWhisk ActionLoop Proxy v%s\n", openwhisk.Version)
+		fmt.Printf("OpenWhisk ActionLoop Proxy v%s, built with %s\n", openwhisk.Version, runtime.Version())
```

Comment:
"built with Go runtime version"

Reply:
Right now I did not mention `Go runtime version` in the string as the string returned by runtime.version() already contains a leading 'go'.
```
OpenWhisk ActionLoop Proxy v1.17.1, built with go1.15.3
```

With the proposed change it will look like this:
```
OpenWhisk ActionLoop Proxy v1.17.1, built with Go runtime version go1.15.3
```
Optionally the double `go` in the output can be avoided:
```
OpenWhisk ActionLoop Proxy v1.17.1, built with go1.15.3 runtime
```
What do you think?
I will then change the message accordingly.

Reply:
my 2 cents is the original message (`OpenWhisk ActionLoop Proxy v1.17.1, built with go1.15.3`) is good.

Reply:
Ok, then I will leave it as it is 👍 .

Reply:
Thanks for the disambiguation of printed output and the string already having 'go` as it was not clear when I commented.